app/sidekiq/event_bus_gateway/va_notify_email_status_callback.rb

```diff
@@ -7,6 +7,22 @@ def self.call(notification)
 
       add_metrics(status)
       add_log(notification) if notification.status != 'delivered'
+      if notification.status == 'temporary-failure' && Flipper.enabled?(:event_bus_gateway_retry_emails)
+        retry_email(notification)
+      end
+    end
+
+    def self.retry_email(notification)
+      ebg_noti = EventBusGatewayNotification.find_by(va_notify_id: notification.id)
+      icn = ebg_noti.user_account.icn
+      profile = MPI::Service.new.find_profile_by_identifier(
+        identifier: icn,
+        identifier_type: MPI::Constants::ICN
+      ).profile
+      EventBusGateway::LetterReadyEmailJob.perform_async(
```

Comment:
Would it be safer to use `#perform_in` with a delay in order to allow some time to pass before retrying?

Reply:
This is a good idea, I support this change if it's possible 👏

Reply:
We could, but remember that this callback is only firing as a result of Notify hitting it, after they've attempted the delivery.